packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -711,12 +737,18 @@ export default abstract class SqlIntegration
       experiment.userIdType
     );
 
+    const ignoreConversionEnd = experiment.attributionModel === "allExposures";
```

Comment:
Right now, all Exposures is just setting `ignoreConversionEnd` but we'll want to refactor this language elsewhere.